packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx

```diff
@@ -82,29 +153,39 @@ export const FocusModeModalHeader: React.FunctionComponent<
   return (
     <div className={controlsContainerStyles}>
       <div className={controlContainerStyles}>
-        <Button
-          size="xsmall"
-          disabled={isFirst}
-          onClick={() => {
-            onStageSelect(stageIndex - 1);
-          }}
-          aria-label="Edit previous stage"
+        <Tooltip
+          isDisabled={isFirst}
+          trigger={({ children, ...props }) => (
+            <span {...props}>
+              {children}
+              <Button
+                size="xsmall"
+                disabled={isFirst}
+                onClick={onPreviousStage}
+                aria-label="Edit previous stage"
+              >
+                <Icon
+                  size="xsmall"
+                  title={null}
+                  role="presentation"
+                  glyph="ChevronLeft"
+                ></Icon>
+              </Button>
+            </span>
+          )}
         >
-          <Icon
-            size="xsmall"
-            title={null}
-            role="presentation"
-            glyph="ChevronLeft"
-          ></Icon>
-        </Button>
-
-        {/* @ts-expect-error leafygreen unresonably expects a labelledby here */}
+          <Body className={tooltipContentStyles}>
+            <span>Goto Previous Stage</span>
```

Comment:
This is label / description, not a title

```suggestion
            <span>Go to previous stage</span>
```